README.md

```diff
@@ -5,6 +5,7 @@ This repository contains the following crates:
 | Name | Description | Links |
 | --- | --- | --- |
```

Comment:
Can we format our markdown to GitHub Flavored format?

Reply:
The table here is already in GFM format~ please check https://github.github.com/gfm/#example-199

Reply:
Ok